website/src/get-started/downloads.md

```diff
@@ -104,6 +104,8 @@ Official [source code download](http://www.apache.org/dyn/closer.cgi/beam/2.7.0/
 
 [Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12343654).
 
+2.7.0 branch is [designated](https://lists.apache.org/thread.html/896cbc9fef2e60f19b466d6b1e12ce1aeda49ce5065a0b1156233f01@%3Cdev.beam.apache.org%3E) by the Beam community as a [long term support (LTS)](https://beam.apache.org/community/policies/) release branch. This branch will be supported for a window of 6 months starting from the day it is marked as an LTS branch ending at 2019-05-16. Beam community will decide on which issues will be backported and when patch releases on the branch will be made on a case by case basis.
```

Comment:
Should this be more discoverable? For example, if we added LTS to the release heading, it would show up in the sidebar nav:

```markdown
 ### 2.7.0 LTS (2018-10-02)
```

Reply:
Yes, I think to so. Added to the heading.